includes/events.js

```diff
@@ -35,6 +35,16 @@ module.exports = (params) => {
                     value: "Contents of the field in the database after it was created or updated, or just before it was imported or destroyed."
                 }
             },
+            hidden_data: {
+                description: "The same as 'data', except dfe-analytics-dataform will attach a policy tag to this field and fields in other tables generated by it to allow it to be masked or hidden from users without permission to access it.",
+                columns: {
```

Comment:
Is 'masked' or 'hidden' the same thing or different? If the same, I might wonder if they are different